constellations/gemini-admin/src/session/session.js

```diff
@@ -0,0 +1,106 @@
+"use strict";
+require("./session.sass");
+import React from "react";
+import API from "../api.js";
+import { SessionAdd } from "./sessionAdd.js";
+import { SessionList } from "./sessionList.js";
+
+export class SessionPage extends React.Component {
+    state = {
+        classes: [],
+        classId: "",
+        sessions: [],
+    };
+
+    componentDidMount = () => {
+        API.get("api/classes/all").then((res) => {
+            const classes = res.data;
+            const classId = classes.length > 0 ? classes[0].classId : "";
+            this.setState({
+                classes: classes,
+                classId: classId,
+            });
+            this.fetchSessionData(this.state.classId);
+        });
+    };
+
+    fetchSessionData = (classId) => {
+        API.get("api/sessions/class/" + classId).then((res) => {
+            var newSessions = res.data;
+            newSessions = _.sortBy(newSessions, ["startsAt"]);
```

Comment:
I'm surprised _.sortBy works. Usually we're supposed to import lodash functions individually. Maybe i botched the library importing... :0